docs/releases.md

```diff
@@ -1,23 +1,45 @@
 ## Releases
 
-## Preparation
-
-To **prepare the extension for a new version**, you must:
-
-- update and commit the version in `package.json`
-- update and commit the `CHANGELOG.md`
-- tag, push, and merge to the `master` branch on GitHub
-
-## Packaging
-
-To **build a signed extension .xpi**, you must: commit and push to the
-`production` branch (ideally as a merge commit from `master`) on GitHub.
-
-## Extension Signing
+### Instructions
+
+1. Update "version" in package.json (and package-lock.json)
+  - we follow the [semver](http://semver.org/) convention
+  - versions "0.1.0" to "1.0.0" is reserved for the internal Alpha
+  - version "1.0.0" is reserved for a public Beta
```

Comment:
I suppose we're not an API (yet?) so semver doesn't matter too much here, but I'm hesitant to say that beta is 1.0, since we may make significant changes before a final (gamma?) release. Do we plan on tagging beta releases as "1.0b1" and such?

Perhaps we could eliminate the numbering distinction between alpha and beta? We could just keep releasing 0.x.y releases until we're out of beta.

Reply:
>Perhaps we could eliminate the numbering distinction between alpha and beta? We could just keep releasing 0.x.y releases until we're out of beta.

That's cool by me.

I was mostly drawing a line in the sand to start and fine with whatever we all decide...

Reply:
This is where I wish alpha was "0.0.z" so that beta could be "0.y.z" and stable "x.y.z".

We really will need something to help differentiate in a report.  If not by number, then by an ending tag (e.g., "0.1.1-apha").

Reply:
>This is where I wish alpha was "0.0.z" so that beta could be "0.y.z" and stable "x.y.z".

We can delete tags and make that call anytime. Whatever you decide is good by me

Reply:
Can't we just make a decision about when we want to be "beta", and then say "0.4.0+ is beta"? That's how I was thinking about doing it for the initial releases.

Reply:
My experience has been an arbitrary value leads to someone getting confused.  This becomes an even bigger problem when there's ongoing experiments or feature testing after beta or stable is released.

Instead of picking which "dot" represents alpha/beta/stable, let's add "-alpha" (or "-beta").